src/languages/lsl.js

```diff
@@ -0,0 +1,90 @@
+    };
+
+    return {
+        illegal: ':',
+        contains: [
+            LSL_STRINGS,
+            {
+                className: 'comment',
+                variants: [
+                    hljs.COMMENT('//', '$'),
+                    hljs.COMMENT('/\\*', '\\*/')
+                ]
+            },
+            LSL_NUMBERS,
+            {
+                className: 'class',
```

Comment:
By the look of the code in the example they may be better served by the 'section' class name. 'Class' is for user class definitions, and they usually have some keywords and a title inside. 'Section' is for sections in structured config files, and here it seems close enough.